src/insanely_fast_whisper/cli.py

```diff
@@ -75,35 +75,22 @@
 def main():
     args = parser.parse_args()
 
-    if args.flash == True:
-        pipe = pipeline(
-            "automatic-speech-recognition",
-            model=args.model_name,
-            torch_dtype=torch.float16,
-            device=f"cuda:{args.device_id}",
-            model_kwargs={"use_flash_attention_2": True},
-        )
-    else:
-        pipe = pipeline(
-            "automatic-speech-recognition",
-            model=args.model_name,
-            torch_dtype=torch.float16,
-            device=f"cuda:{args.device_id}",
-        )
+    pipe = pipeline(
+        "automatic-speech-recognition",
+        model=args.model_name,
+        torch_dtype=torch.float16,
+        device="mps" if args.device_id == "mps" else f"cuda:{args.device_id}",
+        model_kwargs={"use_flash_attention_2": args.flash},
```

Comment:
This is not compatible with mps, is it? Maybe add a hint to the arg that it will be ignored on mps?

Reply:
we could also print a warning when device is mps and flash is enabled